src/repository/opamRepository.ml

```diff
@@ -44,6 +44,11 @@ let cache_file cache_dir checksum =
   in
   aux cache_dir (OpamHash.to_path checksum)
 
+let link_files ~target f l =
+  List.iter (fun x ->
+      OpamFilename.link ~relative:true ~target ~link:(f x))
+    l
```

Comment:
`OpamSystem.link` removes the existing link to create a new one. Do we want that behaviour or check beforehand here if the symlink checksum -> target exists, otherwise create it ?

Reply:
i think we should do this separately in `OpamSystem.link` directly. I've opened https://github.com/ocaml/opam/issues/6113 to track this